src/smartphone.js

```diff
@@ -55,7 +55,7 @@ class Smartphone {
       return (property !== "isAny" && property.indexOf("is") === 0 && typeof Smartphone[property] === 'function');
     });
 
-    for (let index in getAllMethods) {
+    for (let index = 0; index < getAllMethods.length; index++) {
```

Comment:
It would be awesome if the commit had any explanation on *why* this fixes anything. That way people who want to change this line later could find via `git blame` the information about why it's a bad idea. Maybe even adding the explanation as code comments can be worthwhile (again, to avoid someone re-introducing the bug in the future because he/she is unaware of the mootools incompatibility)

Reply:
I added comments